lib/client.rb

```diff
@@ -3,12 +3,13 @@
 require_relative 'redis_tls'
 
 class SentinelClient
-  def initialize(master_name, credentials)
+  def initialize( credentials)
```

Comment:
(cosmetic) can remove extra white-space before `credentials`

Reply:
addressed this change